assembler.ts

```diff
@@ -216,10 +218,10 @@ class Assembler {
     const behaviors: Map<number, RawBehavior> = new Map();
     // Assemble subroutines in reverse call order so parameter rw flags can be propagated
     for (const sub of Array.from(this.subs.values()).reverse()) {
-      behaviors.set(sub.label, this.assembleSub(sub.instructions, behaviors));
+      behaviors.set(sub.label, this.assembleSub(sub.instructions.clone(), behaviors));
```

Comment:
I added cloning because the Code object is manipulated during assembly and it will be visited multiple times in case of nested blueprints/behaviors. A compilation cache might be nice at some point.